docs/sources/static/upgrade-guide.md

```diff
@@ -97,6 +66,35 @@ See [Module and Auth Split Migration](https://github.com/prometheus/snmp_exporte
 The experimental feature Dynamic Configuration has been removed. The use case of dynamic configuration will be replaced
 with [Modules](../../concepts/modules/) in Grafana Agent Flow.
 
+### Breaking change: Removed and renamed tracing metrics
+
+In the traces subsystem for Static mode some metrics are removed and others are renamed.
+The reason for the removal is a bug which caused the metrics to be incorrect if more than one instance of a traces configuration is specified.
+
+Removed metrics:
+- "blackbox_exporter_config_last_reload_success_timestamp_seconds" (gauge)
```

Comment:
These metrics are only removed if the integration is not used

Reply:
Even if the integration is used, would this metric show up? We are using a [detached commit](https://github.com/prometheus/blackbox_exporter/tree/bd22efa1c900a2418e9b5ae3f3d3148f33931b60) of the exporter, and I don't see Agent code calling the [NewSafeConfig](https://github.com/prometheus/blackbox_exporter/blob/bd22efa1c900a2418e9b5ae3f3d3148f33931b60/config/config.go#L92) function anywhere?

Reply:
You are right, those metrics are not relevant to the agent anymore. Good call!